__tests__/Exciter.js

```diff
@@ -133,10 +133,10 @@ test('deleteFailReject', (done) => {
   sinon.stub(exciterAllowReject.dynamo, 'delete').callsFake(awsPromiseReject(new Error('Bad thing happened')));
   return exciterAllowReject.delete(pk, 'fake')
     .then(() => {
-      expect('Promise should reject if rejectOnFail is true').toEqual(false);
+      expect('Promise should reject if rejectOnFail is true').toBe(false);
```

Comment:
There were a lot of unnecessary deep equivalency checks on booleans – hope you don't mind